### DISCUSSION TITLE
Sunbird-coKreat: Release-5.1.3

### DISCUSSION THREAD

**Discussion by Aparna-karri591** [Sunbird-coKreat/Community#50] [General Discussions]
Hello Team,
Release 5.1.3 is In-Progress and here are the details:

|               | Start Date  | End Date |
|------------|:-------------:|:---------------|
| Planning Phase |  16 Nov 2022 | 2 Dec 2022 |
| coKreat Rel 5.1.3.0 Sprint |    16 Jan 2023  | 31 Jan 2023 |
| PPV      | 1 Feb 2023 | 3 Feb 2023 |
| Prod | 6 Feb 2023 | |

**Note**: As Discussed Release 5.1.1 and Release 5.1.2 Tickets will be released as Part of Release 5.1.3

Following are the Features of R 5.1.1,R 5.1.2 and R 5.1.3

Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)

Do engage with us, if you have any questions.
CC:  @vinukumar-vs @chitranshu-keshav @ShailandraOjh @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @ashokreddy1208
@gandham-santhosh
Samagra: @Prerna789 @aayushkedia95

**Comment by vinukumar-vs** [comment]
Follow the sunbird-CoKreat status in the below thread
https://github.com/Sunbird-coKreat/Community/discussions/49
 @ashokreddy1208 @mmanimegalai
